ecommerce/views_test.py

```diff
@@ -1227,6 +1227,27 @@ def test_products_viewset_list(user_drf_client, coupon_product_ids):
         )
 
 
+def test_products_viewset_list_ordering(user_drf_client):
+    """
+    Test that the ProductViewSet returns all products ordered alphabetically
+    by course/program title
+    """
+    response = user_drf_client.get(reverse("products_api-list"))
+    assert response.status_code == status.HTTP_200_OK
+    products = response.json()
+    serialized_products_list = ProductSerializer(data=products, many=True).initial_data
+    ordered_products_title = sorted(
+        serialized_products_list,
+        key=lambda k: k["content_object"]["course"]["title"]
+        if "course" in k["content_object"]
+        else k["content_object"]["title"],
+    )
+    ordered_products_type = sorted(
+        ordered_products_title, key=lambda k: k["product_type"], reverse=True
+    )
+    assert ordered_products_type == serialized_products_list
```

Comment:
Could you please update the test to include program products as well? I expect they would appear on the top of the response

Reply:
The title based ordering of program products is being done at line `#1243` whereas the sort on `#1245` makes sure that `program` type products are on top of the list. Please let me know if you want me to apply it some other way.

Reply:
Okay, looks fine to me, I should've read the test more closely